src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Methodologies/MethodologyService.cs

```diff
@@ -79,6 +85,20 @@ public class MethodologyService : IMethodologyService
                         Owner = false
                     });
 
+                    // If the adopted methodology is unpublished, it may require publishing,
+                    // if the adopting publication is live
+                    var versionToBePublishedId = await _methodologyVersionRepository
+                        .GetMethodologyVersionToBePublished(methodology);
+                    if (versionToBePublishedId != null)
+                    {
+                        var versionToBePublished = _context.MethodologyVersions
+                            .Single(mv => mv.Id == versionToBePublishedId);
+                        versionToBePublished.Published = DateTime.UtcNow;
+                        versionToBePublished.Methodology.LatestPublishedVersionId = versionToBePublished.Id;
```

Comment:
Not really relevant to this PR, but relevant in the sense that it might be something else not working, similar code to this was added in to the Publisher in #4267 

https://github.com/dfe-analytical-services/explore-education-statistics/blob/c31ab4b4ae662407ac405af4eb64552ca1bd9e49/src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/MethodologyService.cs#L64-L76

This code is missing an `Update` on the entity being updated, so it's either not getting updated, or it's relying on something else already calling update elsewhere.

It also replaced a method called `SetPublishedDatesIfApplicable` which has been left unused in the code.